Dockerfile

```
@@ -40,9 +40,10 @@ RUN wget "https://github.com/mozilla/sops/releases/download/v${SOPS_VERSION}/sop
   && sops --version | grep -q "${SOPS_VERSION}"
 
 # Install helmfile
-ARG HELMFILE_VERSION=0.144.0
-RUN wget "https://github.com/roboll/helmfile/releases/download/v${HELMFILE_VERSION}/helmfile_linux_amd64" --quiet --output-document=/usr/local/bin/helmfile \
-  && chmod +x /usr/local/bin/helmfile \
+ARG HELMFILE_VERSION=0.145.0
+RUN wget "https://github.com/helmfile/helmfile/releases/download/v${HELMFILE_VERSION}/helmfile_${HELMFILE_VERSION}_linux_arm64.tar.gz" --quiet --output-document=/tmp/helmfile.tgz \
+  && tar zxf /tmp/helmfile.tgz --strip-components 1 -C /usr/local/bin/ \
```

```suggestion
  && tar zxf /tmp/helmfile.tgz helmfile -C /usr/local/bin/ \
```

- Remove `--strip-components 1` because the archive has all the files at the root (otherwise it removes all top-level files leaving... nothing)
- Adding `helmfile` to ensure that only the binary is copied into `/usr/local/bin` (and not the README/LICENSE file)

---

Your suggestion is better than [mine](https://github.com/jenkins-infra/docker-helmfile/pull/124#discussion_r911038323) ^^